README.rst

```diff
@@ -3,10 +3,22 @@ Tests
 
 Clear Containers Tests Repository
 
-Running functional tests
+Functional tests
```

Comment:
For rst, the underline for a heading must be the same length as the heading itself, so the line below needs to be of length `strlen("Functional tests")` :smile: